bootstrap.py

```diff
@@ -1053,12 +1066,78 @@ def exec_service(service, command, failonerror=True):
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule and options.foreman_fqdn:
```

Comment:
`foreman_fqdn` should be always set at this point, no?

Reply:
Suppose so. In fact, it should throw an exception if it's not.

Reply:
In fact, it already does throw an error if foreman_fqdn isn't set and new_capsule is. Fixed in bec39aa.